ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_statistical.py

```diff
@@ -831,18 +831,10 @@ def test_jax_numpy_std(
 def test_jax_numpy_corrcoef(
     dtype_and_x,
     rowvar,
-    frontend,
```

Comment:
How come these were removed from the test?

Reply:
the last time i resolved the conflict using the web and this apparently caused some line to be removed
today i used pycharm ide and on my fork every thing is perfect